content/applications/websites/website/configuration/google_search_console.rst

```diff
@@ -7,21 +7,175 @@ monitor, maintain, and troubleshoot their site's presence in Google Search resul
+
+.. _GSC-HTML-Tag:
+
+HTML Tag
+--------
+
+Another way to verify ownership is by adding a meta tag to your site's homepage HTML code. Google
+provides you with a unique meta tag that you insert between your homepage's <head> and </head> tags.
+To do so, you need permission to edit the source code of your website’s homepage. After you add your
+website URL under the URL prefix option and click :guilabel:`Continue`, expand the HTML Tag Section
+to find the meta HTML Tag with a personalized key. Copy the tag and paste it into your homepage head
+tag. Make sure not to change the text. Verify that the tag is present on your live page by visiting
+your homepage and checking the page source text to confirm that the tag is present.
+
+Once you complete this step, click :guilabel:`Verify` in Search Console. If you perform the steps
+above correctly, verification should work immediately.
+
+.. _GSC-Google-Analytics-Tracking-Code:
+
+Google Analytics Tracking Code
+------------------------------
+
+If your website already uses Google Analytics, you can verify ownership of your site in Google Search
+Console through your Google Analytics account. This method requires you to be an administrator of
+both Google Analytics and Google Search Console accounts.
+
+.. important:: First, you will need an account with Google Analytics that uses the same Google account.
+   You must have at least edit permission on the Google Analytics account. If you are not sure which
+   permission you have, check the `Google Analytics Help Center <https://support.google.com/analytics/?hl=en#topic=14090456>`_.
+   Before verifying, ensure that your homepage has the Google Analytics tracking code in the head
+   section of the page. Although your Analytics tracking code might work for Analytics in the body
+   section as well, for Search Console verification it must be in the head section.
+
+.. seealso::
+   :doc:`../reporting/analytics`
+
+Open your homepage in a browser, look at the page source code and confirm that the tag is present in
+the head tag of the page.
+
+After you click :guilabel:`Add property`, add your website URL under the URL prefix option and click
+:guilabel:`Continue`.
+If you perform the steps above correctly, verification should work immediately.
+
+.. note:: This works for the verification process only. No data will be collected here, configuring
+   Google Analytics is an entirely separate process.
+
+.. _GSC-Google-Tag-Manager-Container-Snippet:
+
+Google Tag Manager Container Snippet
+------------------------------------
+
+If you use Google Tag Manager to manage tags on your website, you can verify ownership of your site
+by adding a specific HTML tag through the Google Tag Manager interface.
+
+.. _GSC-Google-Sites:
 
-Then, access your Odoo database, go to :menuselection:`Website --> Configuration --> Settings`, and
-enable :guilabel:`Google Search Console` in the :guilabel:`SEO` section. Paste the code in the
-dedicated field.
+Google Sites
```

Comment:
For Google SItes, I recommend removing this section as well because Google Sites is Odoo Website's competitor.
https://sites.google.com/

Reply:
Sorry that I missed that too, thanks for telling I removed it!